core/src/test/scala/integration/kafka/api/SaslSslAdminIntegrationTest.scala

```diff
@@ -633,7 +633,7 @@ class SaslSslAdminIntegrationTest extends BaseAdminIntegrationTest with SaslSetu
   @ValueSource(strings = Array("kraft"))
   def testExpireDelegationToken(quorum: String): Unit = {
     client = createAdminClient
-    val createDelegationTokenOptions = new CreateDelegationTokenOptions()
+    val createDelegationTokenOptions = new CreateDelegationTokenOptions().maxlifeTimeMs(5000)
```

Comment:
why we need this change?

Reply:
This change was made because the previous value of `maxlifeTimeMs` was `5000` milliseconds. Later, I modified the parameter to `Long.MaxValue` to test overflow scenarios. To avoid breaking the original test logic, I changed the `CreateDelegationTokenOptions` that used the default value to explicitly set `.maxlifeTimeMs(5000)`.